gems/pending/util/miq-password.rb

```diff
@@ -191,10 +189,12 @@ def self.ez_load(filename, recent = true)
       CryptString.new(nil, params[:algorithm], params[:key], params[:iv])
     end
   end
+  private_class_method :ez_load
```

Comment:
@kbrock do you have concern about this going from protected to private?  the tests don't fail.  😨